.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] Title of bug"
+labels: ''
```

Comment:
A few things.

`name: Bug Report`

`title: Bug Title`

Can the title remove the `[Bug]` prefix and just specify the label bug instead?